assets/scripts/src/choices.js

```diff
@@ -2198,7 +2202,16 @@ class Choices {
       // Highlight given option, and set accessiblity attributes
       passedEl.classList.add(this.config.classNames.highlightedState);
       passedEl.setAttribute('aria-selected', 'true');
-      this.containerOuter.setAttribute('aria-activedescendant', passedEl.id);
+
+      const hasActiveDropdown = this.dropdown.classList.contains(
+        this.config.classNames.activeState
+      );
+      if (hasActiveDropdown) {
```

Comment:
Quick question - is there a reason why we are setting the aria role inside `_highlightChoice` rather than `showDropdown()`?

Reply:
@jshjohnson have you had time to consider what @vkolmakov1 is talking about, here?
What's your opinion?